### DISCUSSION TITLE
Low performance (in 100 times more) Connect/Bind methods in Docker on Win11 with Linux containers

### DISCUSSION THREAD

**Discussion by P9avel** [dsbenghe/Novell.Directory.Ldap.NETStandard#218] [General]
Hi, i am used simple code

[HttpGet]
        public string Get()
        {
            Stopwatch stopWatch = new Stopwatch();
           
            using (var con = new LdapConnection())
            {
                stopWatch.Start();
                con.Connect("www.zflexldap.com", 389);
                stopWatch.Stop();
            }
            
            TimeSpan ts = stopWatch.Elapsed;
            return ts.ToString();
        }

When i running from VS 2022 i am get 0,2 sec. When running docker file from VS, i am ge 12sec and more (sometime in 100 times more). Please look my image. Can you to help me? What need to do in my case for reducing execution time?

![image](https://user-images.githubusercontent.com/95086762/196059465-49090f43-a2b1-44b4-a5bb-556c7859f194.png)
 
PS. VS 2022 17.3.4, Docker Desktop 4.12.0 (85629), Windows 11, project in atachments 
[TestLDAP.zip](https://github.com/dsbenghe/Novell.Directory.Ldap.NETStandard/files/9795740/TestLDAP.zip)

**Comment by dsbenghe** [comment]
Please first confirm that your setup is correct with one of the linux ldap command line tools

**Reply by P9avel** [reply]
Sorry, i am not understand you. On right side on my screen  am show setting from my docker file. Can you to look again?
![image](https://user-images.githubusercontent.com/95086762/201492346-13f00bce-0da0-4f1a-a713-5acd6702029b.png)

which tools need?
